packages/web-components/src/components/va-link/va-link.css

```diff
@@ -0,0 +1,57 @@
+@import '../../mixins/links.css';
+
+:host {
+  display: block;
```

Comment:
It looks like at least our default links are meant to be inline-block rather than full block.

![Link examples inline](https://user-images.githubusercontent.com/2008881/179063035-b0d7b1f0-4aaf-4c88-a97a-9516a183bb32.png)

And the [guidance at the top says](https://design.va.gov/components/link/)  says:

>  A link is a navigation element that can appear alone, inline (embedded), or in a group with other links.

I don't know if there are any production examples of inline variants like the download link, but we may want to display the component as inline-block.

Reply:
On that note, maybe we should display our links inline in some of the storybook examples.